projects/core/src/site-theme/facade/site-theme.service.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, OnDestroy, inject } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { filter, map, take, tap } from 'rxjs/operators';
+
+import { Observable, Subscription } from 'rxjs';
+
+import { SiteTheme } from '../../model/misc.model';
+import { SiteThemeConfig } from '../config/site-theme-config';
+import { isNotNullable } from '../../util/type-guards';
+import { StateWithSiteTheme } from '../store/state';
+import { SiteThemeSelectors } from '../store/selectors';
+import { SiteThemeActions } from '../store/actions';
+
+@Injectable()
+export class SiteThemeService implements OnDestroy {
+  protected _isInitialized = false;
```

Comment:
```suggestion
  private _isInitialized = false;
```
I really hoe we'll rid it off soon so let's hide it from the customers :)